core/src/main/java/com/scalar/db/api/ConditionalExpression.java

```diff
@@ -19,8 +21,7 @@
  */
 @Immutable
 public class ConditionalExpression {
-  private final String columnName;
-  private final Value<?> value;
+  private final Column<?> column;
```

Comment:
I replaced these `String  columnName` and `Value<?> value` to one field `Column<?> column` here. Maybe we need to discuss it.